packages/components/src/components/button/button.component.ts

```diff
@@ -135,6 +224,12 @@ class Button extends Component {
     }
   }
 
+  /**
+   * Sets or removes the aria-label attribute.
+   *
+   * @param element - The target element.
+   * @param ariaLabel - The aria-label value.
+   */
   private setAriaLabel(element: HTMLElement, ariaLabel: string) {
```

Comment:
this function is completely unnecessary i think, cause if the ariaLabel is passed in, it will be automatically set.